data/summary/summary.go

```diff
@@ -89,10 +102,18 @@ func (c *GlucoseSummarizer[T, A]) GetMigratableUserIDs(ctx context.Context, pagi
 	return c.summaries.GetMigratableUserIDs(ctx, pagination)
 }
 
+func (c *Manager) GetRealtimePatients(ctx context.Context, userIds []string, startTime time.Time, endTime time.Time) (map[string]int, error) {
```

Comment:
The pointer receiver should be changed to `m` and the manager constructor and functions should be moved in this file next to each other.